IBAnimatable/MaskDesignable.swift

```diff
@@ -78,26 +119,30 @@ public extension MaskDesignable where Self: UIView {
       starPoints = 5
     }
     
-    let path = starPath(starPoints)
-    drawPath(path)
+    let path = getStarPath(with: starPoints)
+    draw(path: path)
   }
   
-  private func starPath(_ points: Int, borderWidth: CGFloat = 0) -> UIBezierPath {
+  /**
+   Get a Bezier path for a star shape with provided points.
+   
+   - Parameter sides: The number of the star points.
+   - Returns: A Bezier path for a star shape.
+   */
+  func getStarPath(with points: Int, borderWidth: CGFloat = 0) -> UIBezierPath {
```

Comment:
I don't know if we should document that. It should be private and not usable outside of `IBAnimatable`, so...